### ISSUE TITLE
FileUpload component accessibility issue

### ISSUE THREAD

**Issue by mertsincan** [primefaces/primefaces#3938]
Reported By PRO User;
```
p:fileUpload is coming with the following issue in the accessibility testing

File chooser control does not have an explicit label or a title attribute.

This is how the generated HTML is 
-------------------------------------

<input id="iFamilynetForm:fileUpload_input" name="iFamilynetForm:fileUpload_input" tabindex="-1" type="file"/>
Standard(s) associated with this issue

WCAG 2, Level A, 1.3.1 - Info and Relationships

Section 508, Paragraph N - Electronic Forms

This is the source 
-------------------------------

<p:fileUpload id="fileUpload" value="#{profilePhotoBean.uploadedFile}" mode="advanced" auto="true" allowTypes="/(\.|\/)(jpe?g|png|tiff|bmp)$/"
fileUploadListener="#{profilePhotoBean.handlePhotoUpload}" update="iFamilynetForm:profilePhotoOutputPanel" style="border:1px solid;width:100%"
dragDropSupport="false" sizeLimit="#{profilePhotoBean.maxFileSize}" rendered="#{empty profilePhotoBean.uploadedFile.fileName}"
invalidSizeMessage="#{profilePhotoBean.maxFileSizeExceededMessage}" disabled="#{profilePhotoBean.readOnlyAccess}"
invalidFileMessage="#{profilePhotoBean.invalidFileExtensionMessage}" />
```
